File: VB/WPFBlankDotNETAppWithNorthwindDatabase/Models/Shipper.vb
Imports System
Imports System.Collections.Generic

Namespace WPFBlankDotNETAppWithNorthwindDatabase.Models

	Partial Public Class Shipper
		Public Property ShipperId() As Integer

		Public Property CompanyName() As String

		Public Property Phone() As String

		Public Overridable Property Orders() As ICollection(Of Order) = New List(Of Order)()
	End Class
End Namespace

File: CS/WPFBlankDotNETAppWithNorthwindDatabase/Models/Shipper.cs
using System;
using System.Collections.Generic;

namespace WPFBlankDotNETAppWithNorthwindDatabase.Models;

public partial class Shipper
{
    public int ShipperId { get; set; }

    public string CompanyName { get; set; }

    public string Phone { get; set; }

    public virtual ICollection<Order> Orders { get; set; } = new List<Order>();
}
